install-cloud-sdk-build-task/install-cloud-sdk-build-task.pssproj

```diff
@@ -34,13 +34,13 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="install-cloud-sdk.ps1" />
+    <Compile Include="test\install-cloud-sdk.tests.ps1" />
```

Comment:
Why is it called `.tests.ps1`? It is installing the cloud SDK or testing the install? I know that you are in fact installing the SDK but this doesn't seem like the right name for this. Also, why in the tests directory?

Reply:
It is a Pester unit test file, but you are right that it does not actually test anything. I will remove it for now.